src/content/docs/setup/configuration/commerce-configuration.mdx

```diff
@@ -51,95 +58,381 @@ The `getHeaders` function is a helper function which takes a storefront scope (l
+  - **`Magento-Website-Code`** - Website to connect to. See [Create websites](https://experienceleague.adobe.com/en/docs/commerce-operations/configuration-guide/multi-sites/ms-admin#step-2-create-websites) for details.
+  - **`x-api-key`** - API key for SaaS services (Catalog Service, Live Search, Product Recommendations). See [Commerce Services Connector](https://experienceleague.adobe.com/en/docs/commerce/user-guides/integration-services/saas) for details.
+  - **`Magento-Environment-Id`** - Connects the storefront to the cloud instance serving it. See [Cloud Environment overview](https://experienceleague.adobe.com/en/docs/commerce-cloud-service/user-guide/project/overview#environment-overview) for details.
+
+</TabItem>
+
+<TabItem label="Adobe Commerce as a Cloud Service (ACCS)">
 
-This boilerplate `config.json` for Adobe Commerce requires correct values for your Commerce environment.
+Adobe Commerce as a Cloud Service uses standard Adobe Commerce header naming conventions but does not require API keys or environment IDs in the default configuration.
 
-```json title="config.json" showLineNumbers collapse={20-32} ins={'1':4} ins={'2':5} ins={'3':8} ins={'4':11} ins={'5':12} ins={'6':13} ins={'7':14} ins={'8':15} ins={'9':16}
+```json
+{
+  "headers": {
+    "all": {
+      "Store": "{{STORE_VIEW_CODE}}"
+    },
+    "cs": {
+      "Magento-Store-Code": "{{STORE_CODE}}",
+      "Magento-Store-View-Code": "{{STORE_VIEW_CODE}}",
+      "Magento-Website-Code": "{{WEBSITE_CODE}}"
+    }
+  }
+}
+```
+
+**Header Scopes:**
+
+- **`all`** - Headers applied to all GraphQL requests
+  - **`Store`** - Store view code for Core GraphQL requests. See [GraphQL Headers](https://developer.adobe.com/commerce/webapi/graphql/usage/headers/) for details.
+
+- **`cs`** - Headers for Catalog Service requests
+  - **`Magento-Store-Code`** - Store to connect to. See [Create stores](https://experienceleague.adobe.com/en/docs/commerce-operations/configuration-guide/multi-sites/ms-admin#step-3-create-stores) for details.
+  - **`Magento-Store-View-Code`** - Store view for Catalog Service requests. See [Store views](https://experienceleague.adobe.com/en/docs/commerce-admin/stores-sales/site-store/store-views) for details.
+  - **`Magento-Website-Code`** - Website to connect to. See [Create websites](https://experienceleague.adobe.com/en/docs/commerce-operations/configuration-guide/multi-sites/ms-admin#step-2-create-websites) for details.
+
+</TabItem>
+
+<TabItem label="Adobe Commerce Optimizer (ACO)">
+
+Adobe Commerce Optimizer uses different header naming conventions and requires Commerce Optimizer-specific headers as defined in the [Merchandising Services API documentation](https://developer.adobe.com/commerce/services/optimizer/merchandising-services/using-the-api/#headers).
+
+```json
+{
+  "headers": {
+    "cs": {
+      "AC-View-ID": "{{CATALOG_VIEW_ID}}",
+      "AC-Source-Locale": "{{LOCALE}}",
```

Comment:
I would leave AC-Environment-ID here for now until it has been tested.  It won't hurt to have it, but if it's missing, it can break the configuration.